sdk/include/opentelemetry/sdk/metrics/view/view_registry.h

```diff
@@ -46,29 +46,14 @@ class ViewRegistry
   bool FindViews(
       const opentelemetry::sdk::metrics::InstrumentDescriptor &instrument_descriptor,
       const opentelemetry::sdk::instrumentationscope::InstrumentationScope &instrumentation_scope,
-      nostd::function_ref<bool(const View &)> callback) const
+      nostd::function_ref<bool(const View &)> callback)
   {
-    bool found = false;
-    for (auto const &registered_view : registered_views_)
-    {
-      if (MatchMeter(registered_view->meter_selector_.get(), instrumentation_scope) &&
-          MatchInstrument(registered_view->instrument_selector_.get(), instrument_descriptor))
-      {
-        found = true;
-        if (!callback(*(registered_view->view_.get())))
-        {
-          return false;
-        }
-      }
-    }
-    // return default view if none found;
+    bool found = findViewsInternal(instrument_descriptor, instrumentation_scope, callback);
+    // create default view if none found;
     if (!found)
     {
-      static View view("otel-default-view");
-      if (!callback(view))
-      {
-        return false;
-      }
+      createView(instrument_descriptor, instrumentation_scope);
```

Comment:
Just thinking if we just create a static view with the empty name:
```
if (!found)
{
  static View view("");
  . . . 
}
```

And then this part of code - https://github.com/open-telemetry/opentelemetry-cpp/blob/90fd29183e36377e6c3fa878db8349c8dc7dfd88/sdk/src/metrics/meter.cc#L212 - will not overwrite the name in `InstrumentDescriptor`.

Reply:
@esigo - pinging just in case these comments got missed :)

Reply:
@lalitb thanks, I use static view now.